packages/vscode-pyright/package.json

```
@@ -1763,6 +1763,18 @@
                         }
                     }
                 },
+                "basedpyright.analysis.fileEnumerationMinimumFiles": {
+                    "type": "integer",
+                    "default": 50,
+                    "description": "Minimum number of files that need to be visited before showing a 'slow enumeration' warning. Increase this value to reduce warnings in large workspaces.",
+                    "scope": "resource"
+                },
+                "basedpyright.analysis.fileEnumerationTimeout": {
+                    "type": "integer",
+                    "default": 10,
+                    "description": "Timeout (in seconds) for file enumeration operations. Set to 0 to disable the warning. Increase this value if your workspace is large.",
```

probably wouldn't recommend this just for any large workspace, especially considering that what's considered "large" is subjective. for example my project at work is about 1-2k which i would consider large, but i haven't encountered the issue.

instead we should probably just explain the setting further in the docs, and go into more detail there on when it should / shouldn't be used.
```suggestion
                    "description": "Timeout (in seconds) for file enumeration operations. Set to 0 to disable the warning.",
```